src/utils/index.js

```diff
@@ -129,14 +129,25 @@ var isIOS = module.exports.isIOS = function () {
 };
 
 /**
- * Checks mobile device orientation
- * @return {Boolean} True if landscape orientation
+ * Checks mobile device orientation.
+ * @return {Boolean} True if landscape orientation.
  */
 module.exports.isLandscape = function () {
   return window.orientation === 90 || window.orientation === -90;
 };
 
 /**
+ * Returns whether we can safely listen to this keyboard event.
+ * @returns {Boolean} True if we can.
+ */
+module.exports.isSafeKeyEvent = function (event) {
```

Comment:
safe is not really the word I would use for this function. What about ```isModifierKeyEvent```. Should this be a private function of wasd component?